src/udgl/data-graphics/utils/color-maps.js

```diff
@@ -20,7 +20,7 @@ export function createCatColorMap(options) {
   return function catColorMap(v) {
     const i = getID(v);
     if (i >= 10 || i === undefined) {
-      return 'var(--cool-gray-200)';
+      return 'var(--cool-gray-300)';
```

Comment:
I've gone back and forth on this, but I'm noticing it is actually now pretty hard for my eyes to distinguish the this shade from some of the other colors.  I played with this a bit, and I think for GLAM this works well to make it stand out on hover and also feels just dark enough to be visible + is clearly discernable from the other colors
```suggestion
      return 'var(--cool-gray-100)';
```

Reply:
I merged this, but this doesn't look great. 100 is barely visible over the white background.

Reply:
100 is nice, but I find that it's difficult for me to see with my typical eyesight.

Reply:
100 does look nice in a way, but I find that it's difficult to see against a white background.